misk/web/tabs/loader/src/index.tsx

```diff
@@ -1,3 +1,4 @@
+///<reference types="webpack-env" />
```

Comment:
delete?

Reply:
This is a funky Typescript thing that isn't a comment but is consumed by the Typescript compiler and points it to the correct reference typings.